docs/collectors/PhpFpmCollector.md

```diff
@@ -25,6 +34,9 @@ pm.status_path = /fpm-status
 
 Setting | Default | Description | Type
 --------|---------|-------------|-----
+uri | fpm-status | Path part of the URL, with or without the leading / | string
```

Comment:
What is this table generated from? I've found the above text in docstring but not the table.